addons/pos_hr/static/src/js/Chrome.js

```diff
@@ -20,6 +20,9 @@ odoo.define('pos_hr.chrome', function (require) {
             shouldShowCashControl() {
                 return super.shouldShowCashControl() && this.env.pos.hasLoggedIn;
             }
+            _shouldResetIdleTimer() {
+                return super._shouldResetIdleTimer() && this.tempScreen.name !== 'LoginScreen';
```

Comment:
you introduced a bug, missing to validated the state of the screen.

```js
 && this.tempScreen.isShown

```

Reply:
As you can see the PR is already closed, but you can create a PR if you think there is an issue.

Reply:
@JVegaB 

could you pls?

Reply:
@robinengels I agree with @hugho-ad.

The bug is that the screen doesn't go back to the floorscreen when you are in the product screen. I think we need the condition in pos_hr to be like so:

```js
            _shouldResetIdleTimer() {
                return (
                    super._shouldResetIdleTimer() &&
                    (this.tempScreen.name !== 'LoginScreen' ||
                        (this.tempScreen.name == 'LoginScreen' && !this.tempScreen.isShown))
                );
            }
```

Basically, we should reset the idle timer (restart counting down to automatically change the screen to floorscreen) when:
1. tempScreen is not LoginScreen (the CustomerListScreen)
2. tempScreen is LoginScreen and it's shown (when we are seeing the LoginScreen)

There is a simpler solution though which I think we should do:
```js
        __closeTempScreen() {
            super.__closeTempScreen();
            this.tempScreen.name = null;
        }
```
Can you augment the fix to ensure pos goes back to floorscreen when in product screen?

Reply:
> it also works properly (pr https://github.com/odoo/odoo/pull/91592 was updated)

Nice

> but Why did you use this.tempScreen.name? instead of mainScreen.name?

Unfortunately, the login screen, like the customer list screen, is a tempscreen. It's rendered on top of the main screens (product screen, floor screen, etc).